### PR TITLE
[AIRFLOW-4300] Task Modal Broken for DAGs that have not run

### PR DESCRIPTION
**Pull request by marcusianlevine**
Make sure you have checked _all_ steps below.

### Jira

- [x] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title.
  - https://issues.apache.org/jira/browse/AIRFLOW-4300

### Description

- [x] Here are some details about my PR, including screenshots of any UI changes:

Fixes a bug that prevents the task detail modal from appearing in the Graph view on DAGs that have not yet run

### Tests

- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

Pure JavaScript change to fix a known bug

### Commits

- [x] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [x] In case of new functionality, my PR adds documentation that describes how to use it.
  - All the public functions and the classes in the PR contain docstrings that explain what it does
  - If you implement backwards incompatible changes, please leave a note in the [Updating.md](https://github.com/apache/airflow/blob/master/UPDATING.md) so we can assign it to a appropriate release

### Code Quality

- [x] Passes `flake8`

### CODE REVIEW

**Comment by ashb** [comment]
Thanks! What does the modal look like for this case with your fix?

**Comment by marcusianlevine** [comment]
It looks basically the same way it used to look in older versions, with the current time as the "theoretical" execution date 

Airflow_modal_screenshot

**Comment by ashb** [comment]
It might be worth hiding the "Download Logs by Attempt" and the following `<hr>` if try_number is 0. Could you make that change too?

**Comment by marcusianlevine** [comment]
Sure, I took a stab at hiding that div when `try_number` is 0

I also noticed that the SubDAG link appears to be broken, at least when `try_number == 0`

Going to open a separate issue to follow up on that

File: airflow/www/templates/airflow/dag.html [hunk 1]
```diff
@@ -153,13 +153,13 @@ <h4 class="modal-title" id="myModalLabel">
             View Log
           </a>
           <hr/>
-          <div>
+          <div id="div_dl_logs">
```
Comment (ashb) [inline airflow/www/templates/airflow/dag.html]:
```suggestion
          <div id="dag_dl_logs">
```

**Review by ashb — APPROVED** [review]

**Pull request merged**